app/src/main/java/com/example/firstandroid/presentation/add_bathroom/NewBathroomViewModel.kt

```diff
@@ -42,13 +49,8 @@ class NewBathroomViewModel() : ViewModel() {
         state = state.copy(hours = hours)
     }
 
```

Comment:
Instead of having the scope get created on the view layer, we should create it here in the VM. 

viewmodelScope.launch {} <-- should give you what you need.